pyproject.toml

```diff
@@ -43,7 +43,7 @@ packages = ["fluster", "fluster.decoders"]
 
 [tool.setuptools.data-files]
 "share/fluster/test_suites/aac" = [
-    "test_suites/aac/ISO_IEC_13818-4_2004.json",
```

Comment:
This change here will break BC, but it's necessary to differentiate against suites that we are going to add soon. I think that we can do a new fluster release before the end of the year where we can include all BC breaking code and inform the users in a CHANGELOG about it.
As discussed the other day.
@rsanchez87 @rgonzalezfluendo thoughts?

Reply:
100% agree

Reply:
Totally agree :+1: